app/views/users/show.html.erb

```diff
@@ -1,2 +1,27 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<div class="container m-5">
+  <div class="row">
+    <div class="col-3 border"> 
+      <%= image_tag(@user.photo) %> 
+    </div>
+    <div class="col-8 offset-1 border">
+      <h2> <%= @user.name %> </h2>
+      <h6 class="d-flex justify-content-end"> Number of posts: <%= @user.posts_counter %> </h6>
+    </div>
+  </div>
+
+  <div class="border mt-3 p-2">
+    <p> <%= @user.bio %></p>
+  </div>
+  
+  <% @user.fetch_three_recent_posts.each do |post| %>
+    <div class="border mt-3 p-2">
+      <h3> <%=post.title%> </h3>
+      <p> <%=post.text%> </p>
+      <h6 class="d-flex justify-content-end"> Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter%> </h6>
+    </div>
+  <% end %>
```

Comment:
- [ ] When you click on the post you have to see all the comments that have been commented on each post. Kindly follow the [design](
https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) to implement it.